davdwhyte87/ireporter-webapp
# #164009183 Setup react app

**What does this PR do?**
This PR sets up a barebone react application 

**Tasks to be completed?**
- setup webpack
- configure html loaders
- configure build and start commands
- setup webpack dev server
- set up css loaders and style loaders 

**How can this be manually tested?**
- clone the repo
- install node and yearn
- run yarn install 
- run yarn start and you should see a hello man page 

**Relevant PT story?**
`164009183`
